questdb-rs/src/tests/mod.rs

```diff
@@ -29,6 +29,9 @@ mod http;
 mod mock;
 mod sender;
 
+#[cfg(feature = "ndarray")]
+mod ndarr;
```

Comment:
There should be tests for the trait directly as well, even without `ndarray` enabled.

This is to be sure that we never tie the `ndarray` feature to the ability to write arrays.

The tests that check the `NdArrayView` trait should also check what happens when the trait does nothing or less data.

Reply:
Another test to write is if the trait gives back the wrong number of dimensions.